### PR TITLE
Hannacol/Adding inputs and outputs to the Drill Description page

### PR DESCRIPTION
**Pull request by hannacol**
Drill Description screen was very bland, so we added the inputs and outputs of the drill to the description page.

Also sorted inputs, outputs, and aggOutputs across all screens for consistency

### CODE REVIEW

**Review by solderq35 — APPROVED** [review]
Looks Good To Me.

Some of the Drill Description text could maybe be a little bigger, but not a dealbreaker
image

**Comment by FrankreedX** [comment]
I still think as a user looking at this, this makes no sense... Inputs makes sense, but output is weird. Why is "Carry" outputted when that's my input? Output to where? etc.

File: components/drillDescription.js [hunk 1]
```diff
@@ -28,10 +30,40 @@ export default function DrillScreen(props) {
 
   const windowWidth = Dimensions.get("window").width;
 
+  const sortedInputs = drillInfo.inputs.sort((a, b) => a.id > b.id);
```
Comment (FrankreedX) [inline components/drillDescription.js:33]: Maybe sort this by prettyTitle so when prettyTitle starts with a different letter as the id, it's still sorted correctly. Maybe map it to prettytitle before sorting would be good too.

**Review by FrankreedX — CHANGES_REQUESTED** [review]

**Comment by FrankreedX** [comment]
Maybe add requirements? So the player know what they're hitting against. Kinda like a tldr of the description

**Comment by FrankreedX** [comment]
I'm still not a fan... but if we're rushing, this is good enough

**Review by Gehrkej — COMMENTED** [review]
Everything looks good other than the duplicate description on the modal we discussed.
image

**Review by Gehrkej — APPROVED** [review]
LGTM 👍

**Review by solderq35 — APPROVED** [review]

**Pull request merged**